tests/functional/disaster-recovery/regional-dr/conftest.py

```
@@ -54,7 +54,7 @@ def check_subctl_cli():
         submariner.download_binary()
 
 
-@pytest.fixture(autouse=True, scope="function")
+@pytest.fixture(autouse=False, scope="function")
```

Ruff (0.15.12)

[warning] `scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)